docs/how-to-add-a-new-script-to-the-cea.rst

```diff
@@ -1,16 +1,15 @@
-Contributing
-=============
+How to add a new script
+=======================
 
 So you want to extend the CEA? This guide will get you up and running!
 
 The main steps you need to take are:
 
-#. copy the template script and rename it
-#. update the module-level documentation and credits
-#. develop your script
+#. start with a script template.
+#. develop your script.
 #. add your script to the ``cli.config`` file
```

Comment:
to be changed when the scripts.yml file changes... i can manually update no problem.

Reply:
please do so, thanks!